subsys/mgmt/osdp/src/osdp_phy.c

```diff
@@ -233,7 +233,6 @@ int osdp_phy_packet_finalize(struct osdp_pd *pd, uint8_t *buf,
 		data = is_cp_mode(pd) ? pd->sc.c_mac : pd->sc.r_mac;
 		memcpy(buf + len, data, 4);
 		len += 4;
-	}
```

Comment:
looking at the indenting it looks like it's rather a missing opening curly brace at line 197?

Reply:
you're right! fixed that :)